app/ui/client/views/app/room.js

```diff
@@ -224,6 +224,35 @@ function roomMaxAge(room) {
 	}
 }
 
+async function createFileFromUrl(url) {
+	let response;
+	try {
+		response = await fetch(url);
+	} catch (error) {
+		if (error.toString().includes('TypeError')) {
+			return url;
+		}
+		throw error;
+	}
+	const fileExtension = url.split('.').pop().split(/\#|\?/)[0] || 'jpg';
```

Comment:
Need to improve this chunk to better detect the file extention, maybe even get it from the allowed file types setting